configure.ac

```diff
@@ -120,6 +120,10 @@ GEANY_CHECK_THE_FORCE dnl hehe
 # i18n
 GEANY_I18N
 
+AM_COND_IF([MINGW],
+  [docdir='${prefix}/doc'],
+  [docdir='${docdir}'])
```

Comment:
> This said, I think it'd probably be better to alter paths like this in the Makefile.am instead.

Oh, I see, it's used in src/Makefile.am too.  Well, err hum… so well yeah either not special-case Windows, or indeed to it at configure.